api/bases/cinder.openstack.org_cinderapis.yaml

```diff
@@ -666,6 +666,11 @@ spec:
                 type: object
               transportURLSecret:
                 type: string
+              workers:
+                default: 4
```

Comment:
@ASBishop do you think we need a better default? And also, I'm not sure `minimum: 1` is the right thing to do, we might want to start w/ `4` as minimum value, and scale up.